Makefile

```diff
@@ -68,17 +83,27 @@ build: generate fmt vet ## Build manager binary.
 run: manifests generate fmt vet ## Run a controller from your host.
 	go run ./main.go
 
+.PHONY: container-unit-test
+container-unit-test: ## Build docker image with the manager and execute unit tests.
+	docker build -f Dockerfile --label $(IMAGE_TAG_BASE)-$@:$(VERSION)-$@ -t $(IMAGE_TAG_BASE)-$@:$(VERSION) --load --target testing .
```

Comment:
The `--load` isn't recognized here.
I see it as a buildx command, but not as a build command.

Reply:
This option is used for pushing to a registry.  But here we want to use it out of the local image cache.